clang/include/clang/CIR/MissingFeatures.h

```diff
@@ -230,6 +230,7 @@ struct MissingFeatures {
   static bool emitEmptyRecordCheck() { return false; }
   static bool isPPC_FP128Ty() { return false; }
   static bool emitBinaryAtomicPostHasInvert() { return false; }
+  static bool createLaunderInvariantGroup() { return false; }
```

Comment:
We already have `createInvariantGroup`, should we use that?

Reply:
It seems missing feature  ` createInvariantGroup ` refers to OG's [`DecorateInstructionWithInvariantGroup`](https://github.com/llvm/clangir/blob/a223ec2e5ddf7ebdab521c9fa7916da696b7dee4/clang/lib/CodeGen/CodeGenModule.cpp#L1567). And this is about decorating LLVM IR with invariant group info.

On the other hand, the missing feature `createLaunderInvariantGroup` refers to OG's `[CreateLaunderInvariantGroup](https://github.com/llvm/clangir/blob/a223ec2e5ddf7ebdab521c9fa7916da696b7dee4/llvm/lib/IR/IRBuilder.cpp#L1075)`
Which simply creates a call to intrinsic [`llvm.launder.invariant.group`](https://llvm.org/docs/LangRef.html#llvm-launder-invariant-group-intrinsic). 

Therefore, they are actually different missing features.